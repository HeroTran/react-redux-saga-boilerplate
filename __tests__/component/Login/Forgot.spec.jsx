/*import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import ForgotPassword from '../../../app/renderer/components/AuthUser/ForgotPassword';

let wrapper;
let handleSendEmail;
let handleChangePassword;
let handleResetAllForgot;
beforeEach(() => {
  handleSendEmail = jest.fn();
  handleChangePassword = jest.fn();
  handleResetAllForgot = jest.fn();
  wrapper = (
    <MemoryRouter>
      <ForgotPassword
        isLoading={false}
        isResetSuccess={false}
        isSentCode={false}
        errorSent={false}
        errorReset={false}
        handleSendEmail={handleSendEmail}
        handleChangePassword={handleChangePassword}
        handleResetAllForgot={handleResetAllForgot}
      />
    </MemoryRouter>
  );
});
describe('should be rendered without error', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(wrapper).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
*/