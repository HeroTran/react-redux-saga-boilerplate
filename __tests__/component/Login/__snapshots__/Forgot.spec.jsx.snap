// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be rendered without error matches snapshot 1`] = `
<div
  className="wrapper-forgot"
>
  <div
    className="MuiGrid-container-1"
  >
    <div
      className="MuiGrid-item-2 MuiGrid-grid-xs-12-41 MuiGrid-grid-sm-6-49"
    >
      <div
        className="form-background forgot"
      >
        <div
          className="bg-forgot"
        />
        <div
          className="go-back"
        >
          <a
            href="/login"
            onClick={[Function]}
          />
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-item-2 MuiGrid-grid-xs-12-41 MuiGrid-grid-sm-6-49"
    >
      <div
        className="form-forgot"
      >
        <div
          className="main-forgot"
        >
          <div
            className="logo-forgot"
          />
          <h1
            className="heading-forgot"
          >
            Forgot password?
          </h1>
          <div>
            <p
              className="des-forgot"
            >
              Please send us your email, We will send you a code to reset your password
            </p>
            <form
              onSubmit={[Function]}
            >
              <div
                className="group-form"
              >
                <div
                  className="each-form"
                >
                  <input
                    className="form-control input-form"
                    name="email"
                    placeholder="Email"
                  />
                  <i
                    className="icon icon-email"
                  />
                </div>
                <div
                  className="each-form each-form-center"
                >
                  <button
                    className="btn btn-primary"
                    disabled={false}
                    name="signin"
                    type="submit"
                  >
                    Send
                    
                  </button>
                </div>
              </div>
            </form>
            
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
